source/src/main/java/org/cerberus/database/DatabaseVersioningService.java

```diff
@@ -8394,7 +8394,7 @@ public boolean isDatabaseUptodate() {
                 + "VALUES('PROPERTYTYPE', 'getRawFromXml', 55, 'Get XML code', 'Get XML code');");
 
         // Adding new Element visible Condition.
-        // 1469
+        // 1470
         b = new StringBuilder();
         b.append("INSERT INTO `invariant` (`idname`, `value`, `sort`, `description`, `VeryShortDesc`) VALUES ");
```

Comment:
I would suggest you put sort from 920/930 to 264/265 that way, those conditions will appear right after ifElementPresent / ifElementNotPresent
Appart from that, the rest of the pull request seems very good ;-). Congrats.